python/ray/serve/http_util.py

```diff
@@ -229,4 +229,7 @@ def get_current_servable_instance():
         # Remove endpoints that belong to other class based views.
         serve_cls = getattr(route.endpoint, "_serve_cls", None)
         if serve_cls is not None and serve_cls != cls:
-            routes.remove(route)
+            routes_to_remove.append(route)
+    fastapi_app.routes[:] = [
```

Comment:
Why do you need the indexing here? Could you set it directly (i.e. `fastapi_app.routes = ...`)?

Reply:
fastapi_app disabled attribute assignment. `fastapi_app.routes` will error.